.github/workflows/browserstack.yml

```diff
@@ -21,41 +15,41 @@ env:
 jobs:
   browserstack:
     runs-on: ubuntu-latest
-    if: github.repository == 'Orange-OpenSource/Orange-Boosted-Bootstrap' && (!contains(github.event.commits[0].message, '[ci skip]') && !contains(github.event.commits[0].message, '[skip ci]'))
     timeout-minutes: 30
 
     steps:
       - name: Bail early
-        if: github.event_name == 'pull_request' && (contains(github.event.pull_request.labels.*.name, 'skip:ci') || contains(github.event.pull_request.labels.*.name, 'skip:browserstack'))
         run: exit 0
+        if: github.actor == 'dependabot[bot]'
```

Comment:
If the job can only be launched by pushing _main_ or _v4-dev_ or manually, I don't understand why we should keep this line about Dependabot which can't do those 2 actions.